spec/support/test_setup.rb

```diff
@@ -2,118 +2,54 @@
 class TestSetup
   # A helper class for instantiating a standard set of factories for testing
```

Comment:
```suggestion
  # A helper class for pre-creating test data with FactoryBot factories
  # and maintaining a cache of these objects
  # to avoid the performance problems of
  # repeatedly asking FactoryBot to create trees of objects.
  # Refs: https://evilmartians.com/chronicles/testprof-2-factory-therapy-for-your-ruby-tests-rspec-minitest
```

Reply:
👍